exercises/storeWithArrays.js

```diff
@@ -49,13 +40,19 @@ function newSofa(name, price) {
  */
 function userCanAffordSofa(user, sofa) {
   // This is your job. :)
+  if (user[2] >= sofa[1]) {
```

Comment:
Since `user[2] >= sofa[1]` always evaluates to either `true` or `false`, you can instead write:

```javascript
function userCanAffordSofa(user, sofa) {
  return user[2] >= sofa[1];
}
```